lib/middleware/token.js

```diff
@@ -52,15 +54,15 @@ var TokenError = require('../errors/tokenerror');
  */
 module.exports = function token(server, options) {
   options = options || {};
-  
-  if (!server) { throw new TypeError('oauth2orize.token middleware requires a server argument'); }
-  
+
+  if (!server) { throw new TypeError(g.f('{{oauth2orize.token}} middleware requires a server argument')); }
```

Comment:
It is an argument name! @0candy what do you think? I leave the decision to you guys :-)